src/dmtest/device_mapper/targets.py

```diff
@@ -176,3 +176,24 @@ def __init__(
 class BufioTestTarget(Target):
     def __init__(self, sector_count, dev):
         super().__init__("bufio_test", sector_count, dev)
+
+
+class VDOTarget(Target):
+    def __init__(
+        self,
+        sector_count,
+        dev,
+	physical_blocks,
+	mode,
+	block_map_cache,
+	block_map_period,
+        opts,
```

Comment:
When opened in an editor, it looks like there is inconsistent indentation for parameters physical_blocks, mode, block_map_cache, and block_map_period.